src/components/PositionsList/PositionsList.tsx

```diff
@@ -32,13 +36,16 @@ export const PositionsList: React.FC<IProp> = ({
 }) => {
   const classes = useStyle()
   const history = useHistory()
-  const [page, setPage] = useState(1)
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars-experimental
+  const [defaultPage, setDefaultPage] = useState(initialPage)
```

Comment:
Is it required to define `defaultPage` state?  I mean isn't the `page` state sufficient? I'm asking because you didn't use the `setDefaultPage`